### DISCUSSION TITLE
How to build basic tailwind classes autocomplete

### DISCUSSION THREAD

**Discussion by ASgmbv** [tailwindlabs/tailwindcss#8732] [Help]
Hi everyone,
I am building a little tailwind tool and need to build basic tailwind core classes autocomplete feature.
My first thought was to list all core classes in an array, but it is a very cumbersome process.
Are there are better ways to achieve that? 
Thanks!

**Comment by VigneshVaidyanathan** [comment]
@ASgmbv Did you figure out a way for this?
